Doc/library/datetime.rst

```diff
@@ -213,10 +213,10 @@ A :class:`timedelta` object represents a duration, the difference between two
    Only *days*, *seconds* and *microseconds* are stored internally.
    Arguments are converted to those units:
 
-   * A millisecond is converted to 1000 microseconds.
-   * A minute is converted to 60 seconds.
-   * An hour is converted to 3600 seconds.
-   * A week is converted to 7 days.
+   * A *millisecond* is converted to 1000 *microseconds*.
+   * A *minute* is converted to 60 *seconds*.
+   * An *hour* is converted to 3600 *seconds*.
+   * A *week* is converted to 7 *days*.
```

Comment:
This change does not seem right to me; this list describes unit conversions only.

```suggestion
   * A millisecond is converted to 1000 microseconds.
   * A minute is converted to 60 seconds.
   * An hour is converted to 3600 seconds.
   * A week is converted to 7 days.
```